app/models/client_history/enrollment_view.rb

```diff
@@ -0,0 +1,67 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class ClientHistory::EnrollmentView
```

Comment:
Maybe a comment for what should go in this class? Eg is it for helpers specifically for the `clients/enrollment_table` view?